src/wallet/ismine.h

```diff
@@ -14,7 +14,26 @@
 class CWallet;
 class CScript;
 
-/** IsMine() return codes */
+/**
+ * IsMine() return codes, which depend on ScriptPubKeyMan implementation.
+ * Not every ScriptPubKeyMan covers all types, please refer to
+ * doc/release-notes.md#ismine-semantics for better understanding.
+ * 
+ * For LegacyScriptPubKeyMan,
+ * ISMINE_NO: the scriptPubKey is not in the wallet;
+ * ISMINE_WATCH_ONLY: the scriptPubKey has been imported into the wallet;
+ * ISMINE_SPENDABLE: the scriptPubKey corresponds to an address owned by the wallet user (can spend with the private key);
+ * ISMINE_USED: the scriptPubKey corresponds to a used address owned by the wallet user;
+ * ISMINE_ALL: all ISMINE flags except for USED;
+ * ISMINE_ALL_USED: all ISMINE flags including USED;
+ * ISMINE_ENUM_ELEMENTS: the number of isminetype enum elements.
+ * 
+ * For DescriptorScriptPubKeyMan and future ScriptPubKeyMan,
+ * ISMINE_NO: the scriptPubKey is not in the wallet;
+ * ISMINE_SPENDABLE: the scriptPubKey matches a scriptPubKey in the wallet.
```

Comment:
In commit "doc: Add developer documentation to isminetype" (2400c3aadc2e3c87f5a1eda862bbb45213e9898f)

No need to change and this should all be [deduplicated](https://github.com/bitcoin/bitcoin/blob/fa8dd34e918cd80db215df8937aaf6bb1552fcd9/src/wallet/scriptpubkeyman.cpp#L56) later anyway, but from https://github.com/bitcoin/bitcoin/pull/17350#issuecomment-561388618 it sounded like maybe this should explicitly say ISMINE_SPENDABLE for descriptor wallets doesn't mean spendable